bigtable-client-core-parent/bigtable-client-core/src/main/java/com/google/cloud/bigtable/core/IBulkMutation.java

```diff
@@ -50,12 +52,10 @@
   ApiFuture<Void> add(RowMutation rowMutation);
 
   /**
-   * Performs a {@link IBigtableDataClient#readModifyWriteRowAsync(ReadModifyWriteRow)} on the
-   * {@link ReadModifyWriteRow}. This method may block if
+   * Adds a future task as RPC operation. This method may block if
    * {@link OperationAccountant#registerOperation(ListenableFuture)} blocks.
    *
-   * @param request The {@link ReadModifyWriteRow} to send.
-   * @return a {@link ApiFuture} which can be listened to for completion events.
+   * @param future a {@link ApiFuture} which would be listened for completion events.
    */
-  ApiFuture<Row> readModifyWrite(ReadModifyWriteRow request);
+  void register(final ApiFuture<?> future);
```

Comment:
Please remove `register` and use an `OperationAccountant` in `BigtableBufferedMutatorHelper`

Reply:
Thanks for the suggestion, Have implemented the same.